.github/workflows/ci.yml

```diff
@@ -1,6 +1,10 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - main
```

Comment:
Ohh thanks this has been bugging me for a while

Reply:
Oops, thanks for the reminder! I realize this will affect the CI of the forked repository, which may not be what we are looking for. I modified it to stop E2E tests in forked repositories as they frequently failed, leaving `CI` behind.